packages/fxa-payments-server/src/lib/errors.ts

```diff
@@ -128,12 +129,14 @@ const paymentErrors2 = [
 ];
 
 const paymentErrors3 = ['general-paypal-error'];
+const signupErrors = ['fxa_signup_error'];
 
 cardErrors.forEach((k) => (errorToErrorMessageMap[k] = CARD_ERROR));
 basicErrors.forEach((k) => (errorToErrorMessageMap[k] = BASIC_ERROR));
 paymentErrors1.forEach((k) => (errorToErrorMessageMap[k] = PAYMENT_ERROR_1));
 paymentErrors2.forEach((k) => (errorToErrorMessageMap[k] = PAYMENT_ERROR_2));
 paymentErrors3.forEach((k) => (errorToErrorMessageMap[k] = PAYMENT_ERROR_3));
+signupErrors.forEach((k) => (errorToErrorMessageMap[k] = FXA_SIGNUP_ERROR));
```

Comment:
I know you didn't create this in this PR, but why is this better than directly updating the `errorToErrorMessageMap` when there's a new localizable error message?

Reply:
It's not really better when there's a one to one mapping.